static/src/javascripts/projects/commercial/modules/third-party-tags/lotame.js

```diff
@@ -1,10 +1,53 @@
 // @flow
+import { isInUsOrCa, isInAuOrNz } from 'common/modules/commercial/geo-utils';
 import config from 'lib/config';
-import { isInAuOrNz, isInUsOrCa } from 'common/modules/commercial/geo-utils';
+import { isInTcfv2Test } from 'commercial/modules/cmp/tcfv2-test';
+
+export type LotameData = {
+    ozoneLotameData: Array<string>,
+    ozoneLotameProfileId: string,
+};
+
+let lotameData: LotameData;
+
+const ozoneLotameCallback = (obj) => {
+    console.log("*** ozoneLotameCallback with", obj);
+    lotameData = {
+        ozoneLotameData: obj.getAudiences(),
+        ozoneLotameProfileId: obj.getProfileId(),
+    };
+}
+
+const beforeLoad = () => {
+    console.log("*** BEFORE LOAD");
```

Comment:
👀

Reply:
thought is a useful console until we verify it works end-to-end. It can be removed along with console.log of line 14 once we verify that we receive a response from `onProfileReady`